operator/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -643,6 +658,8 @@ type AdminAPI struct {
 // KafkaAPI configures listener for the Kafka API
 type KafkaAPI struct {
 	Port int `json:"port,omitempty"`
+	// The name of Kafka API endpoint.
+	Name string `json:"name,omitempty"`
```

Comment:
Would `Name is the name of this Kafka listener` be a more appropriate comment here? I'm not sure what "API endpoint" refers to.

Reply:
As this field is retroactively added to the CRD and optional due to `omitempty`, please clarify what the defaulting behavior of this field is in the comment.